ruff.toml

```diff
@@ -0,0 +1,18 @@
+exclude = [
+    "input",
+    "__pycache__",
+    ".venv",
+]
+
+target-version = "py39"
+line-length = 88
+
+[lint]
+select = [
+    "N",  # Enables all pep8-naming rules
+]
+
+[format]
+quote-style = "double"
```

Comment:
is double quotes becoming a standard? I feel I've mostly seen single across our code, so this causes a lot of noise here